internal/browser/browser.go

```diff
@@ -111,6 +112,13 @@ func GetInventoryStatus(ctx context.Context, sku string, locale string) (*Invent
 
 	inventoryStatus := InventoryStatus{}
 
+	if strings.HasPrefix(string(stockResponseBody), "Rate limit exceeded") {
```

Comment:
We don't need to use strings there we can check for the response status code 429 instead.

Reply:
Side note I don't know how happy I am with this logic overall, I'm assuming during the times when stock is active we will end up getting these frequently but if we backup for a second there might be a chance that the user doesn't get a card at all.... thoughts?

Reply:
@ianmarmour Good point about that... stripped out the logic again.